eleventy.config.mjs

```diff
@@ -122,6 +122,10 @@ function basePackage(pkg, stat) {
     platforms: util.cleanPlatforms(release.platforms),
   }))
 
+  const supportsModernSublime = releases.some((release) => {
+    return util.parseSublimeTextMin(release.sublime_text) >= 3000
+  })
```

Comment:
The wording is slightly misleading, but the effect may be fine. You *can* have a single tag series that has worked since ST2 and still works now.

`parseSublimeTextMin` does what it says, but `supportsModernSublime` doesn't mean the same thing. A `parseSublimeTextMax` could do that.

But maybe it's fine? The penalty is just downranking, and I don't know how many ST2+ packages there are that are still well-maintained, even if they work.